### PR TITLE
[14.0][FIX] operating_unit: user access error

### PR DESCRIPTION
**Pull request by Vimal-OSI**

### CODE REVIEW

**Review by AaronHForgeFlow — DISMISSED** [review]

File: operating_unit/models/res_users.py [hunk 1]
```diff
@@ -42,6 +43,7 @@ def _default_operating_units(self):
         comodel_name="operating.unit",
         string="Default Operating Unit",
         default=lambda self: self._default_operating_unit(),
+        domain="[('company_id', '=', current_company_id)]",
```
Comment (AaronHForgeFlow) [inline operating_unit/models/res_users.py:48]: this field maybe should be company dependant?

**Review by osi-scampbell — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
Can you try to make that field company dependant then?

**Comment by max3903** [comment]
@Urvisha-OSI Can you fix that PR please?

**Review by max3903 — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
LG! Can you fix pre-commit? I will merge then

**Comment by Urvisha-OSI** [comment]
> @Urvisha-OSI Can you fix that PR please?

@max3903 Pre-commit issue is fixed in this PR

**Comment by AaronHForgeFlow** [comment]
Can you squash commits? Merge commits are not desirable in the history.

**Comment by AaronHForgeFlow** [comment]
Superseded here https://github.com/OCA/operating-unit/pull/526 and commits squashed

**Pull request closed without merging**